### PR TITLE
[v1.0] Small fix in the README and DENAS NLP demo

### PR DESCRIPTION
**Pull request by tianyil1**
This PR is updated with the followings:

- Small word fix in the general readme.
- Update the DENAS NLP training parameters in the demo.

### CODE REVIEW

**Comment by tianyil1** [comment]
Please help review this PR. Thx. @xuechendi @Jian-Zhang

File: demo/denas/bert/DENAS_BERT_DEMO.ipynb [hunk 1]
```diff
@@ -823,11 +823,11 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.12"
+   "version": "3.9.12 (main, Aug 29 2022, 00:54:58) \n[GCC 11.2.0]"
   },
   "vscode": {
```
Comment (Jian-Zhang) [inline demo/denas/bert/DENAS_BERT_DEMO.ipynb:828]: what's this? @tianyil1

File: demo/denas/bert/DENAS_BERT_DEMO.ipynb [hunk 2]
```diff
@@ -809,9 +809,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python [conda env:pytorch]",
+   "display_name": "pytorch-1.12.0",
    "language": "python",
-   "name": "conda-env-pytorch-py"
+   "name": "python3"
   },
```
Comment (Jian-Zhang) [inline demo/denas/bert/DENAS_BERT_DEMO.ipynb:815]: was this expected?

**Review by Jian-Zhang — COMMENTED** [review]
Comment in line.

File: demo/denas/bert/DENAS_BERT_DEMO.ipynb (hunk 1 above)
Comment (tianyil1) [reply demo/denas/bert/DENAS_BERT_DEMO.ipynb:828]: This is automaticall updated with the local editor, which does not influence the result. @Jian-Zhang

File: demo/denas/bert/DENAS_BERT_DEMO.ipynb (hunk 2 above)
Comment (tianyil1) [reply demo/denas/bert/DENAS_BERT_DEMO.ipynb:815]: This is the same as the above.

**Comment by xuechendi** [comment]
@tianyil1 , @Jian-Zhang , I saw pics in demo page is a quite blue, how about we use white backgroud? 
image

**Comment by xuechendi** [comment]
@tianyil1 , highlight line replace '-recursive' to '--recursive'
image

**Comment by xuechendi** [comment]
@tianyil1 , path change in demo is not changed accordingly
image

**Comment by tianyil1** [comment]
Thanks for your suggestions. I have fixed the above issues. @xuechendi

**Review by xuechendi — APPROVED** [review]
LGTM

**Pull request merged**